src/util/error.js

```diff
@@ -4,6 +4,8 @@ import root from 'window-or-global';
 export default function error (message) {
   const { DEBUG } = root;
   if (!isUndefined(DEBUG) && DEBUG) {
+    console.error(message);
```

Comment:
You suggested some code that was throwing only when DEBUG was true. 
I think we can remove this file and always throw.